### DISCUSSION TITLE
Installing ffmpeg or avconv

### DISCUSSION THREAD

**Discussion by AvrahamTsaban** [oleksis/youtube-dl-gui#112] [Q&A]
Log output:
`2022-11-08 18:34:37,132-DEBUG-Worker_1-ERROR: ffmpeg or avconv not found. Please install one.`
What installing way is the way to have ffmpeg or avconv recognized?

**Comment by oleksis** [comment]
For install and setting FFmpeg read this [comment](https://github.com/oleksis/youtube-dl-gui/issues/61#issuecomment-943506190)
